mcmc_examples/lighthouse/lighthouse.py

```diff
@@ -277,15 +279,17 @@ def run_job(thread_id, model=model, data_points=data_points,
             n_samples=n_samples, seed=seed):
     samples = mcmc_framework.run_mcmc(
         model, data_points, n_samples,
-        degrees_of_freedom=3.0, seed=seed, n_boards=1)  # spinn-3 board
-#        degrees_of_freedom=3.0, seed=seed, n_boards=n_boards)
+        degrees_of_freedom=3.0, seed=seed, n_boards=n_boards)
 
     print('samples: ', samples)
 
+    dirpath = "results_"+strftime("%Y-%m-%d_%H:%M:%S", gmtime()) + "_nboards"\
```

Comment:
Fine as they are but may be even easier to read using f string formnatting.

See; https://docs.python.org/3.6/tutorial/inputoutput.html

Reply:
Thanks, done.